service/src/main/resources/HTMLTemplate.html

```diff
@@ -2126,6 +2130,9 @@ <h3 class="heading heading--two">Annex Documents</h3>
       </dd>
     </div>
   </dl>
+  {% if case_data.userFullName is not empty %}
```

Comment:
Remove if statement. It is always expected to be there. If not there we wanna fail with exception.